README

```diff
@@ -5,4 +5,4 @@ our toolchain in your own projects, please read the Documentation Primer at:
 
 for information about the tools and formatting languages we use.
 Chapter 1 has a Quick Start section, and Chapter 2 covers the software
-and related items in more detail.
+and related items in more detail.
```

Comment:
Oops, that didn't work. What I was suggesting is to revert proposed changes to the README entirely.

The current state of the PR damages the tree by changing the blame for line 9.